packages/Maleo.js/src/server/server.ts

```diff
@@ -57,6 +65,7 @@ export class Server {
       req,
       res,
       dir: this.options.assetDir,
+      isDev: __DEV__,
```

Comment:
`__DEV__` is a webpack generated variables, I think we can use `__DEV__` anywhere right away